hugo.toml

```diff
@@ -15,8 +15,7 @@ titleSeparator = "-"
 [params]
   themeVariant = "cecg"
   customMermaidURL = "https://unpkg.com/mermaid@10.9.1/dist/mermaid.min.js"
-  # if editURL is set, an "Edit" button will be rendered on each page linking to the github source
-#  editURL = "https://github.com/coreeng/core-platform-docs/tree/main/content/"
+  editURL = "https://github.com/coreeng/core-platform-docs/tree/main/content/"
```

Comment:
why enable this?  this will be publicly exposed and the code isn't,

Reply:
it is :)